src/script/externalRoute.ts

```diff
@@ -78,8 +87,10 @@ export const getManageTeamUrl = (utmSource?: string): string => getTeamSettingsU
 
 export const getCreateTeamUrl = (): string =>
   Config.getConfig().FEATURE.ENABLE_ACCOUNT_REGISTRATION && `${Config.getConfig().URL.TEAMS_BASE}/register/email`;
+export const getDecryptError = (): string => getHelpCenterUrl(URL_PATH.DECRYPT_ERROR_1);
```

Comment:
```suggestion
export const getDecryptErrorUrl = (): string => getHelpCenterUrl(URL_PATH.DECRYPT_ERROR_1);
```

Let's make it consistent with the other urls